app/views/user_authentication/recommendations.html.erb

```diff
@@ -0,0 +1,8 @@
+<br>
+<% @current_user.recommendation.each do |burrito| %>
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
+      <%= render "burritos/burrito", burrito: burrito %>
```

Comment:
Since this view partial is named conventionally, you can simplify this to 

```suggestion
      <%= render burrito %>
```
See more here: https://api.rubyonrails.org/classes/ActionView/PartialRenderer.html